tika-parsers/src/main/java/org/apache/tika/parser/html/HtmlHandler.java

```diff
@@ -16,20 +16,6 @@
  */
 package org.apache.tika.parser.html;
 
-import java.io.ByteArrayInputStream;
```

Comment:
If you could turn off your auto-correct for imports or align your imports with ours so that we don't have all these changes, that'd be great!

Reply:
Yes, I saw that too late, sorry :) fixing it now.